front/app/containers/Admin/sideBar/MenuItem.tsx

```diff
@@ -35,7 +35,7 @@ const Text = styled.div`
 
 const MenuItemLink = styled(Link)<{ active: boolean }>`
   flex: 0 0 auto;
-  width: 210px;
+  width: 224px;
   display: flex;
```

Comment:
Only option we had here without renaming the feature in Dutch was to increase the size of the admin sidebar a bit.